data/css/modules/card/_list.scss

```diff
@@ -30,6 +30,14 @@
         font-size: 14px;
     }
 
+    &__content_grid {
+        padding: 20px;
+    }
+
+    &__thumbnail {
+        min-height: 200px;
```

Comment:
I noticed a side-effect of having this value here, when List card is used as navigation card it can't shrink?

![screenshot from 2016-11-02 11-15-44](https://cloud.githubusercontent.com/assets/481418/19932334/7c052bf4-a0ee-11e6-9ceb-40806447b80f.png)